test/integration/cdi/example-driver/deploy/example/pod-external.yaml

```diff
@@ -0,0 +1,40 @@
+# Two pods, one shared, global claim
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: external-claim-parameters
+  namespace: default
+data:
+  a: b
+---
+apiVersion: v1
+kind: ResourceClaim
+metadata:
+  name: external-claim
+spec:
+  resourceClassName: example
+  parameters:
+    apiVersion: v1
+    kind: ConfigMap
+    name: external-claim-parameters
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-external-claim
+spec:
+  restartPolicy: Never
+  containers:
+  - name: with-resource
+    image: ubuntu:22.04
+    command: ["sh", "-c", "ls -la /dev/tty8 ; set |grep envVar2"]
```

Comment:
What's the purpose of `ls -la /dev/tty8`?

Reply:
Why `envVar2`?

Reply:
it's set by POC kubelet plugin. Should we wait for it to be merged first?

Reply:
The POC kubelet plugin shouldn't do that. Let's not merge this here.

Reply:
What it should do then? Which device[s] to inject, which env variables to set?

Reply:
resolved. discussed out of this PR